test/tasks/example/sep/008-opcm-upgrade-stage1/.env

```diff
@@ -0,0 +1,3 @@
+TENDERLY_GAS=15000000
+FORK_BLOCK_NUMBER=8283399
+ETH_RPC_URL="https://sepolia.optimism.io"
```

Comment:
This is an L2 chain ID: 

```
cast chain-id -r  https://sepolia.optimism.io
11155420
```

I wonder if we can just remove the rpc url and block number from this file?